### ISSUE TITLE
Few points to fix in docu

### ISSUE THREAD

**Issue by palmoreck** [Zendro-dev/ZendroStarterPack#14]
Hi there,

Thanks for this awesome project :)

Just a few things that I've found using https://github.com/Zendro-dev/ZendroStarterPack

* Useful but not too dramatic that repo https://github.com/Zendro-dev/graphql-server had an `.env` in its root. 

* Remove `.gitkeep` in https://github.com/Zendro-dev/ZendroStarterPack/tree/master/data_model_definitions as this produces errors when executing `graphql-server-model-codegen` 

* Change number 3 in https://github.com/Zendro-dev/Zendro-dev.github.io/blob/master/setup_data_scheme.md as for Data Model Definition for table B we need a many to many relation:

Data Model Definition for table B:

```
{
    "model": "department",
    "storageType": "mongodb",
    "attributes": {
        "department_id": "String",
        "department_name": "String",
        "employee_ids": "[String]"
    },
    "associations": {
        "employees": {
            "type": "many_to_many", ####change one_to_many to many_to_many
            "implementation": "foreignkeys",
            "reverseAssociation": "departments",
            "target": "employee",
            "targetKey": "department_ids",
            "sourceKey": "employee_ids",
            "keysIn": "department",
            "targetStorageType": "mongodb"
        }
    },
    "internalId": "department_id",
    "id": {
        "name": "department_id",
        "type": "String"
    },
    "useDataLoader": false
}

```

**Comment by coeit** [comment]
Thank you for compiling this issue

### .env file in gql-server
Generally you don't want to commit `.env` files since those are local environt variables that the user specifically has to set himself. There is a note in the starterPack README for that, but I added it to the graphql README
fixed in: [#310583af](https://github.com/Zendro-dev/graphql-server/commit/310583af0d4283cdd5eb299c638c8ffe70042fe1)

### Remove .gitkeep
The problem here is that to have an empty folder commited to a git repository this is the default way to do it. However that leads to the problem you mentioned.

#### Solution
Only process `.json` files in the codegeneration step
fixed in : [#e425338](https://github.com/Zendro-dev/graphql-server-model-codegen/commit/e42533813f1753bfa79f981f2dad555c7004631f)

### many-to-many documentation data-model
That is indeed wrong in the documentation
fixed in: [#8f17cae](https://github.com/Zendro-dev/Zendro-dev.github.io/commit/8f17cae1957d50058f5b7a351ce361648e13d2a9)

**Comment by palmoreck** [comment]
Great.

Thanks @coeit for the support
